src/OVAL/probes/fsdev.c

```diff
@@ -97,37 +94,36 @@ static int is_local_fs(struct mntent *ment)
 		return 0;
 	}
 
-	if (ment->mnt_fsname == NULL) {
-		return 0;
+	const char *fstype = ment->mnt_type;
+	if (oscap_str_startswith(fstype, "fuse.")) {
+		fstype += strlen("fuse.");
+	}
```

Comment:
This is great. I would suggest to add a comment that points to the source of the inspiration, so it could be easily revised later.
Alternatively, we could consider calling the systemd function if it is possible, but that may be too risky, so maybe not.

Reply:
@matejak I have added a comment.